share/spice/time/time.js

```diff
@@ -50,7 +65,11 @@
             day: dateObj.getDate(),
             monthName: months[dateObj.getMonth()],
             year: dateObj.getFullYear(),
-            placeName: chosen.geo.state ? (chosen.geo.name + ", " + chosen.geo.state) : chosen.geo.name,
+            placeName: chosen.overridePlaceName ?
+                chosen.overridePlaceName :
+                chosen.geo.state ?
+                  (chosen.geo.name + ", " + chosen.geo.state) :
+                  chosen.geo.name,
```

Comment:
This ternary is getting a little out of control, maybe assign to a variable above?